sw/device/silicon_creator/lib/drivers/alert_functest.c

```diff
@@ -0,0 +1,103 @@
+// Copyright lowRISC contributors.
```

Comment:
I guess this should be added to `hw/top_earlgrey/dv/verilator_sim_cfg.hjson` and `test/systemtest/config.py`, right?

Reply:
I'll send a follow-up PR to add the tests into our test automation.